python/GafferUI/SpreadsheetUI.py

```diff
@@ -280,56 +282,64 @@ def __init__( self, plug ) :
 
 		with self.__grid :
 
+			self.__tabBar = _ColumnTabBar(
```

Comment:
I think I preferred the left-align of an earlier build. Center looks neater (and can make them nearer to the mouse), but if you have sections with a varying number of tabs, the bar jumps around on screen a lot. If you're new to a sheet, and flicking between tabs to find the column you want, this can make your head explode a little, as the tab widget is often no longer under your mouse!

*Before Click*
![Screenshot 2020-01-20 at 16 10 31](https://user-images.githubusercontent.com/896779/72741488-a4dce400-3b9f-11ea-8758-fd7dc420980b.png)

*After Displacement*
![Screenshot 2020-01-20 at 16 10 35](https://user-images.githubusercontent.com/896779/72741528-ba520e00-3b9f-11ea-8ebd-accc1eb8ffe8.png)

Reply:
This was totally unintentional, and was only broken on Mac. It turns out that it's [impossible to control the alignment from a stylesheet](https://stackoverflow.com/questions/2020935/aligning-qtabbar-from-stylesheets?rq=1), and the default Mac style forces the centre alignment. And it turned out that we used to have a mitigation against this sort of thing, but it has been broken for a while. As discussed, I've opened https://github.com/GafferHQ/gaffer/pull/3578 to fix the problem on master, and we'll live with the misalignment on Mac for the 0.55 series. This will give us a bit more testing time to check there are no unwanted side effects from #3578.